### ISSUE TITLE
different behavior android TV stremio

### ISSUE THREAD

**Issue by lpuglia** [Stremio/stremio-addon-sdk#257]
I'm following the guide to create a series addon: https://stremio.github.io/stremio-addon-guide/step6
My addon works fine on the android and desktop version of streamio, but, the android TV version is refusing to show the content. In particular, if i serve this:
```
 {
          "id": "hiwrld_tt0147753",
          "type": "series",
          "name": "Captain Z-Ro",
          "description": "From his secret laboratory, Captain Z-Ro and his associates use their time machine, the ZX-99, to learn from the past and plan for the future.",
          "releaseInfo": "1955-1956",
          "logo": "https://fanart.tv/fanart/tv/70358/hdtvlogo/captain-z-ro-530995d5e979d.png",
          "imdbRating": 6.9,
          "poster": "https://www.captain-z-ro.com/images/FLYER-PAGE-1_250.gif",
          "background": "https://www.captain-z-ro.com/images/Captain-Z--R0_500.jpg",
          "genres": ["Sci-Fi", "Children", "Educational"],
          "runtime": "15 mins.",
          "videos": [
              { "season": 1, "episode": 1, "id": "hiwrld_tt0147753:1:1", "title": "Christopher Columbus", "released": "1955-12-18" },
              { "season": 1, "episode": 2, "id": "hiwrld_tt0147753:1:2", "title": "Daniel Boone", "released": "1955-12-25" }
          ]
  }
```

this is how it looks like on the desktop version:
![image](https://github.com/Stremio/stremio-addon-sdk/assets/2184440/7508c20e-c344-44cc-8f08-0396e7bc7539)
It's fine, just some broken images, but it if I deploy the app and use the android TV app i get the following screen:
![image](https://github.com/Stremio/stremio-addon-sdk/assets/2184440/f6371743-42d0-47f9-9160-626e64cce61d)
 what is going on?

**Comment by jaruba** [comment]
stremio android tv and stremio web use the new core, the other apps use the old core

the new core is written in rusr and is strict, it will not allow u to set values in other ways than how they are documented in the addon sdk

u need to check the sdk docs and see if u are missing any required fields in meta.videos, for one, i can tell u that we do not set video.released to just a date string

**Comment by lpuglia** [comment]
@jaruba well, changing the release date fixed everything, e.g.:
```
"released": "2012-07-15T18:00:00.000Z"
```
